Code/GraphMol/FileParsers/multithreaded_supplier_catch.cpp

```diff
@@ -104,6 +138,22 @@ TEST_CASE("callbacks SDF") {
       CHECK(mol->getProp<unsigned int>("recordId") == suppl.getLastRecordId());
     }
   }
+  SECTION("readCallbackException") {
+    auto callback = [](const std::string &sdf, unsigned int recordId) {
+      throw std::runtime_error("I'm Sorry Dave. I'm afraid I can't do that.");
+      return std::string(sdf);
+    };
```

Comment:
```suggestion
    auto callback = [](const std::string &, unsigned int ) -> std::string {
      throw std::runtime_error("I'm Sorry Dave. I'm afraid I can't do that.");
    };
```

Reply:
Done; I removed all the unused parameter names; it still builds fine on my Mac. Hopefully it will build now on Linux with warnings-as-errors, too.